api/v1/server/restapi/policy/get_identity_responses.go

```diff
@@ -32,32 +32,35 @@ func NewGetIdentityOK() *GetIdentityOK {
 }
 
 // WithPayload adds the payload to the get identity o k response
-func (o *GetIdentityOK) WithPayload(payload *models.Identity) *GetIdentityOK {
+func (o *GetIdentityOK) WithPayload(payload []*models.Identity) *GetIdentityOK {
 	o.Payload = payload
 	return o
 }
 
 // SetPayload sets the payload to the get identity o k response
-func (o *GetIdentityOK) SetPayload(payload *models.Identity) {
+func (o *GetIdentityOK) SetPayload(payload []*models.Identity) {
 	o.Payload = payload
 }
 
 // WriteResponse to the client
 func (o *GetIdentityOK) WriteResponse(rw http.ResponseWriter, producer runtime.Producer) {
 
 	rw.WriteHeader(200)
-	if o.Payload != nil {
-		payload := o.Payload
-		if err := producer.Produce(rw, payload); err != nil {
-			panic(err) // let the recovery middleware deal with this
-		}
+	payload := o.Payload
+	if payload == nil {
+		payload = make([]*models.Identity, 0, 50)
```

Comment:
Why do we hardcode 50 here? whats the rationale?

Reply:
This code is automatically generated by Swagger - we do not control this